api/v1/sriovnetworknodestate_types.go

```diff
@@ -71,22 +72,48 @@ type InterfaceExt struct {
 type InterfaceExts []InterfaceExt
 
 type VirtualFunction struct {
-	Name       string `json:"name,omitempty"`
-	Mac        string `json:"mac,omitempty"`
-	Assigned   string `json:"assigned,omitempty"`
-	Driver     string `json:"driver,omitempty"`
+	Name            string `json:"name,omitempty"`
+	Mac             string `json:"mac,omitempty"`
+	Assigned        string `json:"assigned,omitempty"`
+	Driver          string `json:"driver,omitempty"`
+	PciAddress      string `json:"pciAddress"`
+	Vendor          string `json:"vendor,omitempty"`
+	DeviceID        string `json:"deviceID,omitempty"`
+	Vlan            int    `json:"Vlan,omitempty"`
+	Mtu             int    `json:"mtu,omitempty"`
+	VfID            int    `json:"vfID"`
+	VdpaType        string `json:"vdpaType,omitempty"`
+	RepresentorName string `json:"representorName,omitempty"`
+}
+
+// Bridges contains list of bridges
+type Bridges struct {
+	OVS []OVSConfigExt `json:"ovs,omitempty"`
+}
+
+type OVSConfigExt struct {
+	// name of the bridge
+	Name string `json:"name"`
+	// bridge-level configuration for the bridge
+	Bridge OVSBridgeConfig `json:"bridge,omitempty"`
+	// uplink-level bridge configuration for each uplink(PF).
+	// currently must contain only one element
+	Uplinks []OVSUplinkConfigExt `json:"uplinks,omitempty"`
+}
+
+type OVSUplinkConfigExt struct {
+	// pci address of the PF
 	PciAddress string `json:"pciAddress"`
```

Comment:
will this always exist even if a bond is connected to the ovs?

Reply:
yes, in this case we will have two or more entries in the OVSCofnigExt.uplinks, each entry represents a single interface